06_create_cluster.sh

```diff
@@ -58,6 +58,11 @@ if [[ -n "${APPLY_EXTRA_WORKERS}" ]]; then
     fi
 fi
 
+# Create a secret containing extraworkers info for the e2e tests
```

Comment:
Same question about `OPENSHIFT_CI` below also is in here.

Reply:
If it isn't very resource intensive then I'd add it regardless,
if we do decide to only add it in CI then we should make sure that the error message from openshift-tests makes it clear whats missing (for anybody trying to reproduce tests manually)

Reply:
Yep, looks something that we could keep it anyway